00-basics/002-javascript/freeCodeCamp_03_18.js

```diff
@@ -0,0 +1,331 @@
+function testGreaterOrEqual(val) {
+    if (val >= 20) {
+        return "20 or Over";
+    }
+    if (val >=10) {
+        return "10 or Over";
+    }
+    return "Less than 10";
+}
+console.log(testGreaterOrEqual(0));
+
+function testLessThan(val) {
+    if (val < 25) {
+        return "Under 25";
+    }
+    if (val < 55) {
+        return "Under 55";
+    }
+    return "55 or Over";
+}
+console.log(testLessThan(35));
+
+function testLessOrEqual(val) {
+    if (val <= 12) {
+        return "Smaller Than or Equal to 12";
+    }
+    if (val <= 24) {
+        return "Smaller Than or Equal to 24";
+    }
+    return "More than 24";
+}
+console.log(testLessOrEqual(25));
+
+function testLogicalAnd(val) {
+    if (val <= 50 && val >= 25){
+        return "Yes";
+    }
+    return "No";
+}
+console.log(testLogicalAnd(10));
+
+function testLogicalOr(val) {
+    if (val < 10 || val > 20) {
+        return "Outside";
+    }
+    return "Inside";
+}
+console.log(testLogicalOr(15));
+
+function testElse(val) {
+    let result = "";
+    if (val > 5 ) {
+        result = "Bigger than 5";
+    } else {
+        result = "5 or Smaller";
+    }
+return result;
+}
+console.log(testElse(4));
+
+function  testElseIf(val) {
+    if (val > 10) {
+        return "Greater than 10";
+    } else if (val < 5) {
+        return "Smaller than 5";
+    }
+    else {
+        return "Between 5 and 10";
+    }
+}
+console.log(testElseIf(7));
+
+function orderMyLogic(val) {
+    if (val < 5) {
+        return "Less than 5";
+    } else if (val < 10) {
+        return "Less than 10";
+    } else {
+        return "Greater than or equal to 10";
+    }
+}
+console.log(orderMyLogic(7));
+
+function testSize(num) {
+    if (num < 5) {
+        return "Tiny";
+    } else if (num < 10) {
+        return "Small";
+    } else if (num < 15) {
+        return "Medium";
+    } else if (num < 20) {
+        return "Large";
+    } else if (num >= 20){
+        return "Huge";
+    }
+    return "Change Me";
+}console.log(testSize(7));
+
+let names = ["Hole-in-one!", "Eagle", "Birdie", "Par", "Bogey", "Double Bogey", "Go Home!"];
+function golfScore(par, strokes) {
+    if (strokes === 1) {
+        return names [0];
+    } else if (strokes <= par -2) {
+        return names [1];
+    } else if (strokes === par) {
+        return names [3];
+    } else if (strokes === par + 1) {
+        return names [4];
+    } else if (strokes === par + 2) {
+        return names [5];
+    } else if (strokes >= par +3) {
+        return names [6];
+    }
+    return "Change Me";
+}
+console.log(golfScore(5,4));
+
+
+function caseInSwitch(val) {
+    let answer = "";
+    switch(val) {
+        case 1:
+            return "alpha";
+            break;
+        case 2:
+            return "beta";
+            break;
+        case 3:
+            return "gamma";
+            break;
+        case 4:
+            return "delta";
+            break;
+     //Why Unreachable code? all break (181, 184, 187, 190 )
```

Comment:
When return statement (e. g. `return "alpha"`) is invoked, execution of the switch ends, making explicit stopping of switch execution (by invoking `break` statement) redundant.

Note that if your code looked like this:
```
    switch(val) {
        case 1:
            console.log("alpha");
            break;
        case 2:
            console.log("beta");
            break;
        case 3:
            console.log("gamma");
            break;
        case 4:
            console.log("delta");
            break;
```

then removing all `break` would result in not just specific `case` executing, but also all of the cases below it as well (so called `switch fall-through` - we keep going down, executing case after case starting from the one matched by `case` value, until interrupted by `break`)